src/rgw/rgw_common.cc

```diff
@@ -166,6 +166,8 @@ void req_info::rebuild_from(req_info& src)
   effective_uri.clear();
   host = src.host;
 
+  env->set("CONTENT_TYPE", src.env->get("CONTENT_TYPE"));
```

Comment:
What if there is no Content-type header?
(some admin command don't have content type header)
Please check before setting it.